### ISSUE TITLE
docker: an error in the systemctl daemon log

### ISSUE THREAD

**Issue by eatradish** [AOSC-Dev/aosc-os-abbs#1695]
Bug description
-----------------------
```
Mar 01 17:18:45 eat-x220i dockerd[7425]: Failed to start containerd: exec: "containerd": executable file not found in $PATH
```
more log: telegram @ShimizuSaki

I try update docker to 18.09.3 and `ln -s /usr/bin/docker-containerd /usr/bin/containerd`, and after:
https://pastebin.aosc.io/paste/b~xLHAAAyWHmDAHtdnC2Lw

Reproducing steps
---------------------------

systemctl start docker

Update channel(s)
--------------------------

Testing and Explosive

Package and version
------------------------------
```
 ~  apt show docker                                                                                                                    600ms  Fri Mar  1 19:46:01 2019
Package: docker
Version: 18.09.3
Section: admin
Maintainer: sakiiily <sakiiily@aosc.io>
Installed-Size: 237 MB
Depends: lxc (>= 3.1.0), bridge-utils (>= 1.6), btrfs-progs (>= 4.20.1), iproute2 (>= 4.18.0), sqlite (>= 3.27.1-1), systemd (>= 1:240-2), libseccomp (>= 2.3.3)
Download-Size: 47.6 MB
APT-Sources: https://repo.aosc.io/debs testing/main amd64 Packages
Description: Automates deployment of containerized applications

N: There are 7 additional records. Please use the '-a' switch to see them.
```
and 18.09.2

**Comment by eatradish** [comment]
https://git.archlinux.org/svntogit/community.git/commit/trunk/PKGBUILD?h=packages/docker&id=bf36fd563dccc22c15cf6b872d0faa97ea864603
maybe fix this issue

**Comment by eatradish** [comment]
fix of ` 6a22ca0`, close this issue.
